### ISSUE TITLE
WrfProj does not have attributes ll_lat and ll_lon. vertcross fails because of this.

### ISSUE THREAD

**Issue by weech** [NCAR/wrf-python#44]
I am trying to use vertcross with a pivot point that is a lat, lon pair. This fails, and the bottom of the stack trace is at interputils.to_xy_coords. In interputils.to_xy_coords, the call of _ll_to_xy references WrfProj attributes ll_lat and ll_lon, but these are never defined anywhere. I tried manually adding ll_lat and ll_lon attributes to my projection object using the reference latitude and longitude in my WPS namelist, but this lead to the function returning an incorrect cross section (the latitude was all wrong and it didn't follow the 90° pivot angle I gave it, which is another issue). I don't understand enough about projections to know how this would be fixed.

**Comment by bladwig1** [comment]
Can you please provide the stack trace?

**Comment by bladwig1** [comment]
I'm also going to need to see how you're trying to call the function, because this works here on my system:


```
from wrf import CoordPair, vertcross, getvar
from netCDF4 import Dataset as nc
wrfin = nc("wrfout_d02_2005-08-28_00:00:00")
pivot = CoordPair(lat=25, lon=-86.5)
p = getvar(wrfin, "pressure")
z = getvar(wrfin, "z")
v = vertcross(p, z, wrfin=wrfin, pivot_point=pivot, angle=90.0)
```

Result:
```
<xarray.DataArray u'pressure_cross' (vertical: 100, cross_line_idx: 96)>
array([[       nan,        nan,        nan, ...,        nan,        nan,
               nan],
       [981.82764 , 981.8374  , 981.84064 , ..., 984.30066 , 984.6592  ,
        984.9832  ],
       [959.5765  , 959.5954  , 959.5967  , ..., 961.8662  , 962.21826 ,
        962.52954 ],
       ...,
       [ 58.349968,  58.363743,  58.376606, ...,  58.20668 ,  58.212433,
         58.21818 ],
       [ 56.364464,  56.378555,  56.391735, ...,  56.242287,  56.247917,
         56.2535  ],
       [ 54.37898 ,  54.393387,  54.406883, ...,        nan,        nan,
               nan]], dtype=float32)
Coordinates:
    XTIME     float32 0.0
    Time      datetime64[ns] 2005-08-28
    xy_loc    (cross_line_idx) object CoordPair(x=0.0, y=45.0, lat=25.022436142, lon=-90.5740585327) ...
  * vertical  (vertical) float32 0.0 204.98376 409.96753 614.9513 819.93506 ...
Dimensions without coordinates: cross_line_idx
Attributes:
    FieldType:      104
    description:    pressure
    units:          hPa
    stagger:        
    coordinates:    XLONG XLAT XTIME
    projection:     Mercator(stand_lon=-89.0, moad_cen_lat=27.9999923706, tru...
    orientation:    (0.0, 45.0) to (95.0, 45.0) ; center=CoordPair(lat=25, lo...
    missing_value:  9.96920996839e+36
    _FillValue:     9.96920996839e+36

```

**Comment by weech** [comment]
Yes, it is (edited for privacy)
```python
Traceback (most recent call last):
  File "theta.py", line 155, in <module>
    main()
  File "theta.py", line 52, in main
    pivot_point=pivot, angle=90)
  File "[home]/miniconda3/lib/python3.6/site-packages/wrf/metadecorators.py", line 1602, in func_wrapper
    return _set_cross_meta(wrapped, instance, args, kwargs)
  File "[home]/miniconda3/lib/python3.6/site-packages/wrf/metadecorators.py", line 866, in _set_cross_meta
    stagger, projection)
  File "[home]/miniconda3/lib/python3.6/site-packages/wrf/interputils.py", line 417, in to_xy_coords
    ref_lat=projection.ll_lat, 
AttributeError: 'LambertConformal' object has no attribute 'll_lat'
```
The function in my script that sets it off reads
`wrf.vertcross(potential_temp, height, levels=vlevs, latlon=True, projection=proj, pivot_point=pivot, angle=90)`
where proj is the projection pulled from `height.attrs['projection']` and pivot is `wrf.CoordPair(lon=-122.3321, lat=47.6062)`.

**Comment by bladwig1** [comment]
Yeah, this is a bug. I'll see if I can sneak in a fix for the 1.1 release later this week.
